c7n/resources/rds.py

```diff
@@ -832,6 +832,51 @@ def set_retention_window(self, resource, retention, copy_tags):
             CopyTagsToSnapshot=copy_tags)
 
 
+@actions.register('set-public-access')
+class RDSSetPublicAvailability(BaseAction):
+    """
+    This action allows for toggling an RDS instance
+    'PubliclyAccessible' flag to true or false
+
+    :example:
+
+        .. code-block: yaml
+
+            policies:
+              - name: enable-rds-public-accessibility
+                resource: rds
+                filters:
+                  - PubliclyAccessible: false
```

Comment:
can you switch the example to the case of remediating a public database, thats going to be the 99% use case here.